index.html

```diff
@@ -258,17 +258,69 @@ <h3><a> Let's connect </a> </h3>
                   </ul>
                 </li>
                 <li class="frameworks">
-                  <p class="">Frameworks</p>
+                  <h2> Frameworks </h2>
                 </li>
                 <li class="skills">
                   <h2> Skills </h2>
                 </li>
               </ul>
             </div>
-    </section>
-
-
+          </section>
 
+          <section class="contact__me__section">
+         <div class="contact__me">
+           <h3 class="contact__me__title">Contact Me</h3>
+           <p class="contact__me__text">
+               If you have an application you are interested in developing,
+               a feature that you need built or a project that needs coding.
+               I’d love to help with it
+           </p>
+           <form
+             class="contact__me__form"
+             action="https://formspree.io/f/mjvlknzp"
+             method="post"
+             id="form"
+           >
+             <input
+               class="input"
+               type="text"
+               id="name"
+               name="full-name"
+               maxlength="30"
+               placeholder="Name"
+               required
+             />
+             <input
+               class="input"
+               type="email"
+               id="email"
+               name="email"
+               placeholder="example@gmail.com"
+               required
+             />
+             <textarea
+               class="text__area"
+               name="message"
+               id="textArea"
+               cols="30"
+               rows="10"
+               maxlength="500"
+               placeholder="Write your message here ..."
+               required
+             >
+           </textarea>
```

Comment:
- [ ] Let's make sure we don't have spaces between text area and it's closing tag because that counts as value of textarea which affects user experience while writing you a message `<textarea ..></textarea>` as shown below
![image](https://user-images.githubusercontent.com/87689547/155786158-cace065b-77b1-470a-bc22-a1ed93b86a26.png)